Doc/using/cmdline.rst

```diff
@@ -302,8 +302,9 @@ Miscellaneous options
    Run Python in isolated mode. This also implies :option:`-E`, :option:`-P`
    and :option:`-s` options.
 
-   In isolated mode :data:`sys.path` contains neither the script's directory nor
-   the user's site-packages directory. All :envvar:`PYTHON*` environment
+   In isolated mode, :data:`sys.path` does not contain the current working
+   directory, the script's directory, or the user's site-packages directory.
```

Comment:
You may add something like:

```
(...): see the :option:`-P option <-P>` for details.
```

Reply:
I think this is already indicated in L302~L303. Wouldn't this be a duplication?

In summary:

- `the current working directory, the script's directory` comes from `-P`
- `the user's site-packages directory` comes from `-s`
- ``All :envvar:`PYTHON*` environment`` comes from `-E`

I am also wondering how to make it informative while being concise. 😂

Anyway, the point is, if `the script's directory` is mentioned, `the current working directory` should also be mentioned. Otherwise, they should all appear in the `see ... for details`.